CHANGELOG.md

```diff
@@ -13,11 +13,15 @@ Documentation for HIPIFY is available at
 * Full `hipSOLVER` support
 * Full `rocSPARSE` support
 * New options:
-  * `--amap` to hipify as much as possible, ignoring '--default-preprocessor' behavior
+  * `--amap` to hipify as much as possible, ignoring `--default-preprocessor` behavior
+  * `--clang-resource-directory` to specify the clang resource path - the path to the parent folder for the `include` folder,
+    containing `__clang_cuda_runtime_wrapper.h` and other header files used during hipification process
 
 ### Fixes
 
-* Do not rewrite tokens in code blocks skipped by Preprocessor (under the '--default-preprocessor' option)
+* Code blocks skipped by the Preprocessor are not hipified anymore under the `--default-preprocessor` option
+* Clang resource files used during hipification are being searched now and can also be specified
+  by the `--clang-resource-directory` option
```

Comment:
Is this a continuation of the previous bullet point or a new one? @emankov

Reply:
The first bullet was rephrased, the second bullet is about the new fix (since previous changes of CHANGELOG for 6.1), which I hope will be cherry-picked into 6.1 RC3.

Reply:
Thanks, Evigeny. Would the following edit work? 

Clang resource files used during hipification are now searchable and can be specified by the `--clang-resource-directory` option.

Reply:
Clang resource files used during hipification are now searchable and **also** can be specified by the --clang-resource-directory option.

Reply:
I am okay with your version since this is a changelog entry. We must reword it if it's a highlight in the release notes.